features/step_eight.feature

```diff
@@ -59,7 +59,7 @@ Feature: Step eight page
   Scenario: No income
     Given I am a single person on step eight
     When I select no income
-    Then I am taken to step eleven
+    Then I am taken to step '11''probate'
```

Comment:
`(\d+) ([^"]*)` should pick up the `Then I am taken to step 11 probate`
and
`'(\d+) ([^"]*)'` should pick up the `Then I am taken to step '11 probate'`

Any of the above is fine. Personally, I would go with the first one - less characters, easier to read.